### PR TITLE
GIVCAMP-24 | Release workflow

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW

# Summary
- Auto release workflow

# Review Tasks

## Setup tasks and/or behavior to test

1. Look at code?

# Associated Issues and/or People
- JIRA ticket(s) - GIVCAMP-24

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        id: release
+        uses: release-drafter/release-drafter@v5
+        with:
+          version: ${{ env.RELEASE_TAG }}
+          publish: true
+        env:
+          GITHUB_TOKEN: ${{ secrets.ADMIN_GITHUB_TOKEN }}
+
+      - name: Merge release back into dev
+        uses: devmasx/merge-branch@v1.4.0
```
Comment (yvonnetangsu) [inline .github/workflows/release.yml:98]: I pretty much stole the one from SAA homesite but updated the version of merge-branch to v1.4.0. Also, removed the reference to the fontawesome license token.

**Comment by yvonnetangsu** [comment]
@sherakama Does this look legit for an autorelease workflow? Copy pasta from SAA repo with a couple changes in the comment above 🤔

**Review by sherakama — APPROVED** [review]
This looks good.

If you run in to errors you may have to check/update the github action secret but I think you're good to go there. 

One other thing to note, the release version is based off of the version in package.json so you will be starting at > 0.0.1, which is fine, but something to know.

**Comment by yvonnetangsu** [comment]
> This looks good.
> 
> If you run in to errors you may have to check/update the github action secret but I think you're good to go there.
> 
> One other thing to note, the release version is based off of the version in package.json so you will be starting at > 0.0.1, which is fine, but something to know.

Thanks @sherakama ! What would your recommendation be for the starting version in this case? 😃

**Comment by sherakama** [comment]
0.0.1 is fine by me. 
Some feel strongly it should be 1.0.0-beta|alpha|rc 

I like to save the version 1.0.0 release for go-live. 

Your choice.

**Comment by yvonnetangsu** [comment]
> 0.0.1 is fine by me. Some feel strongly it should be 1.0.0-beta|alpha|rc
> 
> I like to save the version 1.0.0 release for go-live.
> 
> Your choice.

Yup - that sounds like a good plan to me! ⭐

**Pull request merged**